lib/src/main/java/com/auth0/msg/InvalidClaimsException.java

```diff
@@ -0,0 +1,14 @@
+package com.auth0.msg;
+
+/**
+ * A runtime exception that is thrown when there is an invalid claim in a Message object type
+ */
+public class InvalidClaimsException extends RuntimeException {
```

Comment:
Extending RuntimeException is not right. You should extend Exception. This should be a checked exception so that caller can handle and recover from this.

Reply:
Done